.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Descent 3 Build
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+
+      matrix:
+        os: [windows-latest]
+        build_type: [Debug, Release]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
I would suggest using the [modern `checkout?v4`](https://github.com/actions/checkout) based on a supported Node version, instead of the legacy v3.

That is unless there is a particular reason that I’m missing completely 🤗

Reply:
Oh, no reason at all, just copied what I had around in some other project 🙂. I'll change it to v4